### PR TITLE
Format SIG tables

### PR DESCRIPTION
**Pull request by trask**
(note: GitHub's "rich diff" option does a better job of rendering the differences)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,47 +123,47 @@ SIG-specific GitHub discussions.
...
-Community Demo Application|Every Monday at 8:30 PT|[Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit)|[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA)|
-Functions as a Service (FAAS) |Every other Tuesday at 12:00 PM PT |[Google Doc](https://docs.google.com/document/d/187XYoQcXQ9JxS_5v2wvZ0NEysaJ02xoOYNXj08pT0zc/edit#)|[Slack](https://cloud-native.slack.com/archives/C04HVBETC9Z)|
-OpenTelemetry Profiling WG | Every other week on Thursday at 08:00 AM PT |[Google Doc](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C03J794L0BV)|
-Security | Weekly on Wednesday at 08:30 AM PT |[Google Doc](https://docs.google.com/document/d/1P2xejC7lEkOV_Z-8E0oZPXLK5HOnUPNuRqKP0ZQ5fpg/edit?usp=sharing)|[Slack](https://cloud-native.slack.com/archives/C05A85QC281)|
-Event WG|Every Friday at 10:00 AM PT|[Google Doc](https://docs.google.com/document/d/1BKjQWP32FXL9g1cGbyj7DMXV1Uq_RL8_78rWaMBhN0A/edit)|[Slack](https://cloud-native.slack.com/archives/C062HUREGUV)|
```
Comment (trask) [inline README.md:166]: @jack-berg @tigrannajaryan I moved Event WG up under Specification SIGs, let me know if that's incorrect, thx

File: README.md [hunk 2]
```diff
@@ -123,47 +123,47 @@ SIG-specific GitHub discussions.
...
+| Semantic Conventions: General        | Every Monday at 08:00 PT             | [Google Doc](https://docs.google.com/document/d/10xG7DNKWRhxNmFGt3yYd3980a9uwS8lMl2LvQL3VNK8)      | [#otel-semantic-conventions-wg](https://cloud-native.slack.com/archives/C041APFBYQP) | [Armin Ruech](https://github.com/arminru)<br/>[Josh Suereth](https://github.com/jsuereth)<br/>[Reiley Yang](https://github.com/reyang) |
+| Semantic Conventions: Messaging      | Every Thursday at 8:00 PT            | [Google Doc](https://docs.google.com/document/d/1dWHhyXnfVife-cQ2DW5-d5Ldp1Lq8Rre2UsHpyo8cEE/)     | [#otel-messaging](https://cloud-native.slack.com/archives/C02Q4AAHDSA)               |                                                                                                                                        |
```
Comment (trask) [inline README.md:134]: changed the link here, it used to point to the `#otel-instrumentation` channel

**Review by svrnm — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (jack-berg) [reply README.md:166]: That seems correct to me

File: README.md [hunk 3]
```diff
@@ -123,47 +123,47 @@ SIG-specific GitHub discussions.
...
+| Ruby: SDK                             | Every Tuesday at 09:00 PT                                                                                  | [Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)                        | [#otel-ruby](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions) |
+| Rust: SDK                             | Every Tuesday at 16:00 UTC                                                                                 | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             | [#otel-rust](https://cloud-native.slack.com/archives/C03GDP0H023) and [Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)                          |
```
Comment (arminru) [inline README.md:154]: @open-telemetry/rust-maintainers is Gitter still in use? The last message there was sent 8 months ago. You could consider archiving it in favor of Slack and removing the link here in a follow-up PR.

**Review by arminru — APPROVED** [review]
Thank you!

**Pull request merged**